pkg/version/version.go

```diff
@@ -13,6 +13,9 @@ var DdevVersion = "v0.3.0-dev" // Note that this is overridden by make
 // for examples defining version constraints.
 var DockerVersionConstraint = ">= 17.05.0-ce"
```

Comment:
These should both be const, not var right?

Reply:
Everything in that file is `var`, so I stayed consistent.

Reply:
Totally understood. Was just giving you the option of making it better if you want to. This code has a vast lineage, and sometimes we're ambitious enough to improve, sometimes we go with the flow. You went with the flow. As I would in my first week! Was just giving you options.